enacts/calc.py

```diff
@@ -596,6 +596,51 @@ def _cess_date(dry_thresh, dry_spell_length_thresh, sm_func, time_coord):
 # Time functions
 
 
+def interval_to_point(interval, to_point="mid", keep_attrs=True):
+    """ Create an xr.DataArray of the left, mid or right points of an Interval
+    dimension
+
+    Parameters
+    ----------
+    interval : xr.DataArray(pd.Interval)
+        array of intervals
+    to_point : str, optional
+        "left", "mid" or "right" point of `interval`
+        default is "mid"
+    keep_attrs : boolean, optional
+        Keep attributed from `interval` to point array
+        default is True
+
+    Returns
+    -------
+    point_array : xr.DataArray
+        array of the left, mid or right points of `interval`
+
+    See Also
+    --------
+    pandas.Interval
+
+    Notes
+    -----
+    Should work for any type of array, not just time.
+    xr.groupby_bins against dim renames the Interval dim_bins,
+    not sure if xr.groupby does the same,
+    and what other Xarray functions return Intervals but, depending,
+    could generalize the returned array name
+    """
+    return xr.DataArray(
+        data = [getattr(interval.values[t], to_point) for t in range(interval.size)],
```

Comment:
PEP8 says no spaces around the equals sign when it's used in a kwarg.

(PEP8 is the style guide for contributions to python itself, but is also generally used for python programs in general.)